test/conftest.py

```diff
@@ -99,6 +99,7 @@ def mock_publication(test_session):
             doi="10.1515/9781503624153",
             sulpub_json={"sulpub": "data"},
             wos_json={"wos": "data"},
+            dim_json={"dimensions": "data"},
```

Comment:
i wanted some starting fixture data that was different than what i'd expect to get written by the test, so that i could implicitly test that the field got updated by checking for the value written by the test.